docs/shared/core-features/automate-updating-dependencies.md

```diff
@@ -20,11 +28,7 @@ First, run the `migrate` command:
 nx migrate latest # same as nx migrate nx@latest
 ```
 
-You can also specify the name of the package and the version:
-
-```shell
-nx migrate nx@version # you can also specify version
-```
+Note you can also specify an exact version by replacing `latest` with `nx@<version>`.
```

Comment:
Do we want to mention that you can also update just a single third-party plugin with `nx migrate my-plugin@version`?

Reply:
yeah good idea, I'll add that